app/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ plugins {
     id("jacoco")
     alias(libs.plugins.nowinandroid.android.application.firebase)
     id("com.google.android.gms.oss-licenses-plugin")
+    alias(libs.plugins.dependencyGuard)
```

Comment:
Should it be part of the Android Application convention plugin?

Reply:
I had thought about that briefly. The only other app is the catalog, and that one literally is just UI, so I'm not sure how valuable that would be. What do you think?

Reply:
Added to the Android Application convention plugin